src/Measures.jl

```diff
@@ -11,7 +11,8 @@ export discretize_values,
 	get_total_correlation,
 	get_interaction_information,
 	get_partial_information_decomposition,
-	get_redundancy
+	get_redundancy,
+	cross_entropy,
```

Comment:
Could you rename it to get_cross_entropy? These long names do need to be shortened (see #9), but we should keep them consistent now and shorten them all at once.
Remove trailing comma too :)